docs/profiling.md

```diff
@@ -0,0 +1,31 @@
+# vHive profiling tool
+
+## Methodology
+
+The benchmark loads a certain amount of requests per second (RPS) to vHive for a time duration.
```

Comment:
please explain the purpose(s) of this tool. What can it be used for?

then, you need to explain the setup: loader goroutines and a single latency measurement goroutine, and so on

Reply:
Done